### ISSUE TITLE
npx source-css does not generate different values

### ISSUE THREAD

**Issue by Telewa** [namad/sds-bootstrapper#1]
Given 

// source.config.json
```
{
    "hue": "190",
    "saturation": "0.2",
    "distance": "0.02",
    "primary": "pink",
    "info": "teal",
    "success": "green",
    "warning": "amber",
    "danger": "red",
    "red": "4",
    "amber": "25",
    "brown": "33",
    "green": "130",
    "teal": "184",
    "blue": "210",
    "indigo": "240",
    "violet": "260",
    "purple": "280",
    "pink": "340",
    "typeScale": "minorThird",
    "accentSaturation": "0.9",
    "accentMaxLuminance": "0.6",
    "accentMidLuminance": "0.18",
    "accentMinLuminance": "0.1"
}
```
npx source-css ./src/css/variables.css

generates values with 
```
...
    --primary-100: var(--accent-blue-100);
    --primary-200: var(--accent-blue-200);
    --primary-300: var(--accent-blue-300);
    --primary-400: var(--accent-blue-400);
    --primary-500: var(--accent-blue-500);
    --primary-600: var(--accent-blue-600);
...
```

instead of

```
...
    --primary-100: var(--accent-pink-100);
    --primary-200: var(--accent-pink-200);
    --primary-300: var(--accent-pink-300);
    --primary-400: var(--accent-pink-400);
    --primary-500: var(--accent-pink-500);
    --primary-600: var(--accent-pink-600);
...
```

is this expected behavior?

**Comment by namad** [comment]
Fix is ready, new package 1.0.3 has been released.

Thanks!

**Comment by Telewa** [comment]
Thank you so much!
